### DISCUSSION TITLE
How to select deck to which the new term will be exported?

### DISCUSSION THREAD

**Discussion by homocomputeris** [arianneorpilla/jidoujisho#296] [Q&A]
How to select deck to which the new term will be exported?

**Comment by brishtibheja** [comment]
![Screenshot_2023-08-30-20-46-20-27_d1e85489dfcf960faafc2f4d5db41573](https://github.com/lrorpilla/jidoujisho/assets/136738526/71a5a8bd-294e-42ce-93b2-3b71d2cd3f7c)
See the top line in card browser view. My deck says 新イマージョンデッキ📺

**Comment by helica1** [comment]
Isn't it this little triangle?

[image: image.png]
